src/index.coffee

```diff
@@ -47,26 +47,32 @@ class DatadogPlugin
     stats = statsObject.report()
 
     metrics =
-      'scenarios.created': stats.scenariosCreated
-      'scenarios.completed': stats.scenariosCompleted
-      'requests.completed': stats.requestsCompleted
-      'response.2xx': 0
-      'response.3xx': 0
-      'response.4xx': 0
-      'response.5xx': 0
+      'scenarios.created': [stats.scenariosCreated, datadog.increment]
+      'scenarios.completed': [stats.scenariosCompleted, datadog.increment]
+      'requests.completed': [stats.requestsCompleted, datadog.increment]
+      'requests.pending': [stats.pendingRequests, datadog.increment]
+      'response.2xx': [0, datadog.increment]
+      'response.3xx': [0, datadog.increment]
+      'response.4xx': [0, datadog.increment]
+      'response.5xx': [0, datadog.increment]
```

Comment:
all these metrics are actually counts, and hence we should treat them as datadog metric type count instead of gauge. it doesn't matter when running from one machine, but becomes important when used via serverless-artillery which uses multiple lambdas. these metrics then do not get combined if the metrics are treated as type gauge instead of type count.